libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Common/SFKeychainItemWrapper.h

```diff
@@ -110,29 +99,25 @@ extern NSString * _Nullable const kSFKeychainItemExceptionErrorCodeKey;
  */
 - (BOOL)resetKeychainItem;
 
-/* Passcode related methods */
-
 /**
- Sets the passcode in the keychain.
- @param passcode Plain text passcode
- */
-- (void)setPasscode:(nullable NSString *)passcode;
-
-/** The passcode.
+ Store arbitrary data to the keychain for the service (identifier) and account specified in the initializer.
+ @param data Arbitrary data to store to in the keychain. May be `nil`.
+ @return The status of the keychain update request.
  */
-- (nullable NSString *)passcode;
+- (OSStatus)setValueData:(nullable NSData *)data;
 
-/** Performs passcode verification.
- @param passcode The passcode to verify.
+/**
+ Read arbitrary string from the keychain for the service (identifier) and account specified in the initializer.
+ @return Arbitrary string read from the keychain. May be `nil`.
  */
-- (BOOL)verifyPasscode:(nullable NSString *)passcode;
+- (nullable NSString *)valueString;
```

Comment:
Is it possible to switch `valueString` to a `property` instead, with a `setter`?

Reply:
It's not a new method. Just one I'm exposing through the header.